# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/ffmpeg/FFmpegRuntimeLinker.h
# Commit: e3f920da2232
# Full Hash: e3f920da22320921473f385cb060ed7978d67df9
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P3: Only create PDMs that are supported in the current process. r=kamidphish
#   
#   Depends on D52797
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D54876
# ==============================================================================

diff -r 44471f99280c -r e3f920da2232 dom/media/platforms/ffmpeg/FFmpegRuntimeLinker.h
--- a/dom/media/platforms/ffmpeg/FFmpegRuntimeLinker.h	Tue Oct 20 23:23:58 2020 +0000
+++ b/dom/media/platforms/ffmpeg/FFmpegRuntimeLinker.h	Tue Oct 20 23:24:27 2020 +0000
@@ -14,7 +14,7 @@
 class FFmpegRuntimeLinker {
  public:
   static bool Init();
-  static already_AddRefed<PlatformDecoderModule> CreateDecoderModule();
+  static already_AddRefed<PlatformDecoderModule> Create();
   enum LinkStatus {
     LinkStatus_INIT = 0,   // Never been linked.
     LinkStatus_SUCCEEDED,  // Found a usable library.